### PR TITLE
Test direct pram access

### PR DESCRIPTION
**Pull request by martijnvandermarel**
Added capacity_test GES
Tested EtherCAT capacity with 40-byte input and 40-byte output succesfully
Tested new slave step-by-step with @rarriens 
Changed ESI files based on new default SM addresses

### CODE REVIEW

File: ges/CapacityTest_GES/src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  miso.a = 1;
+  miso.b = 2;
+  miso.c = 3;
+  miso.d = 4;
+  miso.e = 5;
+  miso.f = 6;
+  miso.g = 7;
+  miso.h = 8;
+  miso.i = 9;
+  miso.j = 10;
```
Comment (Ishadijcks) [inline ges/CapacityTest_GES/src/main.cpp]: Is it somehow possible to loop through miso?

File: ges/CapacityTest_GES/src/main.cpp [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    miso.a = mosi.A;
+    miso.b = mosi.B;
+    miso.c = mosi.C;
+    miso.d = mosi.D;
+    miso.e = mosi.E;
+    miso.f = mosi.F;
+    miso.g = mosi.G;
+    miso.h = mosi.H;
+    miso.i = mosi.I;
+    miso.j = mosi.J;
```
Comment (Ishadijcks) [inline ges/CapacityTest_GES/src/main.cpp]: Isolate to a function, see above comment

File: ges/CapacityTest_GES/src/utypes.h [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      uint32_t A;
+      uint32_t B;
+      uint32_t C;
+      uint32_t D;
+      uint32_t E;
+      uint32_t F;
+      uint32_t G;
+      uint32_t H;
+      uint32_t I;
+      uint32_t J;
```
Comment (Ishadijcks) [inline ges/CapacityTest_GES/src/utypes.h]: Can't you use arrays of some sort?

Comment (martijnvandermarel) [reply ges/CapacityTest_GES/src/utypes.h]: done

File: ges/Example_GES/src/main.cpp [hunk 4]
```diff
@@ -12,14 +17,18 @@
 #define miso            Ethercat::pdoTx.Struct.miso
 #define mosi            Ethercat::pdoRx.Struct.mosi
 
+// Set PDO sizes
+const int PDORX_size = 40;
```
Comment (rarriens) [inline ges/Example_GES/src/main.cpp]: Why these sizes?

Comment (martijnvandermarel) [reply ges/Example_GES/src/main.cpp]: Because array of 10 32-bit numbers = 320 bits with 8 bits/byte = 40 bytes.

**Comment by martijnvandermarel** [comment]
Closes #17

**Pull request merged**